backend/drafts.js

```diff
@@ -6,41 +6,35 @@ function setupDraftsRoute(app) {
   app.post('/api/drafts', async (req, res) => {
     const { subject_id } = req.body;
     try {
-      const result = await db.query('INSERT INTO drafts (subject_id) VALUES (?)', [subject_id]);
-      res.status(201).json({ id: result.insertId });
+        const result = await db.query('INSERT INTO drafts (subject_id) VALUES (?)', [subject_id]);
+        res.status(201).json({ id: result[0].insertId });
     } catch (err) {
-      res.status(500).json({ error: 'Error creating draft' });
+        res.status(500).json({ error: 'Error creating draft' });
     }
   });
 
 
-  app.get('/api/drafts/:id', async (req, res) => {
-    const { id } = req.params;
+  app.get('/api/drafts', async (req, res) => {
     try {
-      const [rows] = await db.query('SELECT drafts.id, drafts.subject_id, inspection_information.* FROM drafts JOIN inspection_information ON drafts.id = inspection_information.draft_id WHERE drafts.id = ?', [id]);
-      if (rows.length === 0) {
-        res.status(404).json({ error: 'Draft not found' });
-      } else {
-        res.json(rows[0]);
-      }
+        const result = await db.query('SELECT * FROM drafts');
+        res.status(200).json(result);
     } catch (err) {
-      res.status(500).json({ error: 'Error retrieving draft' });
+        res.status(500).json({ error: 'Error fetching drafts' });
     }
   });
 
 
-  app.put('/api/drafts/:id', async (req, res) => {

```

Comment:
After consulting the AI I decided that changing a draft id doesn't make a whole lot of sense as it would a bunch of tangling references to the other tables. Also, the API was missing get api/drafts/id route, so I asked the ChatGPT to add that.